### PR TITLE
ISLANDORA-897: Add scrolling to More Advanced Controls

### PR DESCRIPTION
**Pull request by bondjimbond**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-897)

# What does this Pull Request do?

Allows scrolling in the blocks within More Advanced Controls. 

# What's new?
If there are multiple values in a given window, enough to go beyond the bounds of the box, you could not scroll through them to see what's outside the bounds. Now you can, using the mouse's scrolling feature. This also solves a problem in Chrome where selecting lower-down items with the arrow keys would not result in the window scrolling.

# How should this be tested?

* Create a form element with a field type that requires values in More Advanced Controls, e.g. "select" type
* Under More Advanced Controls, put 8 to 10 different values in the Options box
* Try scrolling, note that it doesn't work
* Check out this branch -- scrolling is better!

# Additional Notes:
Scrolling still isn't perfect, but it's better than it was. Would love to add an actual scroll bar, but that doesn't seem to work, possibly due to other, earlier code or just the way it's built. This does the job at least.

# Interested parties
@d-r-p @DiegoPino  @Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by d-r-p** [comment]
@bondjimbond I am starting to think that this could be a platform (OS) issue. Unfortunately, I only have two UNIX flavours to test on (and no mouse wheel), and on neither I can see a difference after checking out your branch - neither in FF nor Chromium (the latter only on one system). On both branches I see a scroll bar, but no scrolling when moving to the upper or lower edge (so I have to move items window size by window size - tedious, but gets the job done).

I guess this needs a different tester. Sorry...

**Comment by bondjimbond** [comment]
@d-r-p You may be right; I just tested on a Surface Pro (Windows), and scrolling works. But it doesn't on my Macbook. Can you report whether my changes have any impact on your scrolling? If there's no difference, I'd take that as a good sign - improving things for Mac users without harming Windows users.

**Comment by d-r-p** [comment]
@bondjimbond I have no way on testing this on Win/Mac. On two UNIX flavours, however, there seems to be no difference (at least in FF).

**Comment by rosiel** [comment]
I've seen this issue too, sporadically. I was using chrome on a mac. But I suspect there's something else going on (A JavaScript thing not loading? A browser plugin?) Because it doesn't happen all the time.

**Comment by bondjimbond** [comment]
@rosiel Nothing to do with plugins, as far as I can tell. I've tested on multiple browsers on Mac, with and without extensions enabled; scrolling in those windows is always a problem. 

Adding the lines I add in this PR provides scrolling functions in those scenarios. 

@rosiel Are you able to replicate the scrolling problem on your Mac now, given the testing steps I wrote up? @d-r-p detects no problems on Unix, so we know it doesn't hurt Unix users... If we can confirm that (a) it helps the Mac users who are affected, and (b) doesn't hurt anyone else, is that Good Enough?

**Review by DonRichards — APPROVED** [review]
Worked!

**Comment by DonRichards** [comment]
Final Product.

**Comment by DiegoPino** [comment]
@DonRichards worked on how many browser versions/brands?

**Comment by DonRichards** [comment]
@DiegoPino I didn't check. The description specified chrome but now I wonder if this breaks anything for other browsers. Looking now.

**Comment by DonRichards** [comment]
Firefox & Safari render fine. 
Firefox
Safari

**Comment by DiegoPino** [comment]
@DonRichards thanks for checking!

**Comment by DonRichards** [comment]
Windows 10 Edge browser. 
download 2

**Comment by DonRichards** [comment]
I feel like this is fine and I still approve this for merging unless someone else has an issue with their browser.

**Comment by bondjimbond** [comment]
Thanks for testing, @DonRichards!

**Comment by bondjimbond** [comment]
@DonRichards Just reminding you that this is waiting for your merge.

**Pull request merged**
